backend/dal/Migrations/v01.06.03/Down/PostDown/01-Buildings.sql

```diff
@@ -0,0 +1,9 @@
+PRINT 'Update Buildings'
```

Comment:
So these allow the project number to be maintained throughout the migration. during Up the non-null project number is just written to the projectnumbers as an array. On the Down, the first project number in the array is saved in the projectNumber field.